napari/layers/_image_layer/_constants.py

```diff
@@ -0,0 +1,28 @@
+from enum import auto
+
+from ...util.misc import StringEnum
+
+
+class Interpolation(StringEnum):
```

Comment:
do you think this Enum needs its own file?

Reply:
I like the constants having their own files for consistencies sake, and the fact that we may add more constants here too. Maybe some of the colormap processing stuff should be in here too instead of the model file. Thoughts @jni?

Reply:
I agree that we should be consistent across the layer types. Given that the other layers store things like Enum's in a `_constants.py` file, I think we should do the same here. As @sofroniewn mentioned, we may add other things here in the future anyway.